SilKit/source/core/internal/traits/SilKitMsgTraits.hpp

```diff
@@ -126,81 +126,86 @@ struct SilKitMsgTraits
     };
 
 DefineSilKitMsgTrait_TypeName(SilKit::Services::Logging, LogMsg) DefineSilKitMsgTrait_TypeName(
```

Comment:
I'm going to create a separate PR to fix the messy formatting of this file in particular.

Reply:
The PR exists now here: https://github.com/vectorgrp/sil-kit/pull/86